### ISSUE TITLE
About Host Application installation and Uniquify function

### ISSUE THREAD

**Issue by minshifu** [bobbyrne01/save-text-to-file-firefox#105]
Thank you for your addon. It is really frustrating that I've tried many times to install the Host Application,strictly following your instructions but failed every time. Could you kindly provide a video demo? Besides, does the Uniquify function means I can append new text to the original file? Best regards!

**Comment by bobbyrne01** [comment]
@minshifu I made a video demo of installing the host application, please see [here](https://www.youtube.com/watch?v=o0kgMorHmdc).

**Comment by minshifu** [comment]
Thank you so much. I finally made it. In the meanwhile I also found out what went wrong with my installation. I directly downloaded the needed files from this page https://github.com/bobbyrne01/save-text-to-file-firefox/tree/master/app    Thus the formats were all wrong. Besides, successful installation was achieved only after I restarted my computer. I don't why. Anyway I finally made it. Many thanks!
